### DISCUSSION TITLE
是否可以有个选项启用或者关闭打印的 Start Crawling/finish 信息

### DISCUSSION THREAD

**Discussion by cdwmhcc** [coder-hxl/x-crawl#64] [Q&A]
如果可以手动配置就更好了，类似 https://listr2.kilic.dev/task/output.html

**Comment by coder-hxl** [comment]
自己控制启动或关闭爬取记录也是个不错想法，麻烦您在 https://github.com/coder-hxl/x-crawl/issues/new/choose 提一下新想法建议 😄
